arangod/Graph/Graph.h

```diff
@@ -70,7 +81,7 @@ class EdgeDefinition {
   /// types of values.
   static Result validateEdgeDefinition(const velocypack::Slice& edgeDefinition);
 
-  static ResultT<EdgeDefinition> createFromVelocypack(velocypack::Slice edgeDefinition);
+  static ResultT<EdgeDefinition> createFromVelocypack(velocypack::Slice edgeDefinition, std::set<std::string> const& satCollections);
```

Comment:
```suggestion
  static ResultT<EdgeDefinition> createFromVelocypack(velocypack::Slice edgeDefinition, std::unordered_set<std::string> const& satCollections);
```

I think sortedness is not required.

Reply:
True, but I would like to keep it sorted here to have sorted velocypack output (we're doing this for edgeColls, vertexColls as well)